doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -15,50 +17,69 @@ First, you need to run the Node-RED by using the following command in terminal:
 
 >Note: instead of changing **Output** parameter to `a parsed JSON object`, you can also use a `JSON` node or the `function` node with proper parsing function.
 
-Next, add two `debug` nodes to the flow and connect both of them to the `mqtt in` node. One debug node will listen to the events, while the other will listen to messages coming from the devices. To make this happen, set the **Output** parameter for the first debug node configuration to `complete msg object` value, and to `msg.payload` for the other. 
+Next, add two **debug** nodes to the flow and connect both of them to the **mqtt in** node. One debug node will listen to the events, while the other will listen to messages coming from the devices. To make this happen, set the **Output** parameter for the first debug node configuration to `complete msg object` value, and to `msg.payload` for the other. 
+
+{{< figure src="receive_uplink_flow.png" alt="Final flow scheme" >}}
+
+After setup, click on **Deploy** in the upper right corner. If the setup is correct, below the **mqtt in** node `connected` status will be reported. In upper righ corner, click on debug icon to see the event messages and their payloads in JSON format.
+
+## Send Messages 
+
+This section follows the process of setting up a flow which publishes messages to a certain topic that MQTT Server is subscribed to. By doing this, you are scheduling downlink messages to be sent to your end device. This section follows the example for publishing downlink traffic in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide.
+
+Find the **mqtt out** node in Node-RED library and place it on the dashboard. This node connects to MQTT broker and publishes messages. Configure the **Server** option same as in the upper section. As stated in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide, topic `v3/{application id}/devices/{device id}/down/push` can be used for scheduling downlink messages. Choose a **QoS** from listed options and state whether you want the MQTT Server to retain messages. 
+
+{{< figure src="mqtt_out_node_properties.png" alt="mqtt out node properties" >}}
+
+Further, find and place the **inject** node on the dashboard. This node injects a message into a flow and can be triggered manually, but it is also possible to define regular intervals between the automatic injections. Double-click on the **inject** node to configure its properties. **Payload** field can have different types of format - in this example JSON format is being used. Expand the **Payload** field and enter this:
+
+```json
+{
+  "downlinks": [{
+    "f_port": 15,
+    "frm_payload": "vu8=",
+    "priority": "NORMAL"
+  }]
+}
+```
+
+in order to schedule a downlink message with hexadecimal payload `BE EF`. State the period between the automatic injections if you want them or choose `None` for **Repeat** if you do not. 
```

Comment:
Inconsistent use of `code block` vs **literal** to be found in the screen vs "value" to be entered

Reply:
Could you please clarify this?

Reply:
- Use `code blocks` for code, file names, configuration keys, environment variables, etc, all the technical identifiers
- Use **strong** for text that you can find literally in the user interface, like **Gateways** or **User name** in the login form
- Use "quotes" for input

Here, you say `None`, while that should be "None"

I'm also open to suggestions for changing this, as long as we are consistent. cc @benolayinka

Reply:
The word 'none' here is actually not a value to be entered (although it looks like that because of the small case), it's being chosen from the dropdown list. I guess that means that it should be **none**, to be consistent with **a parsed JSON object**

Reply:
> * Use `code blocks` for code, file names, configuration keys, environment variables, etc, all the technical identifiers
> * Use **strong** for text that you can find literally in the user interface, like **Gateways** or **User name** in the login form
> * Use "quotes" for input
> 
> Here, you say `None`, while that should be "None"
> 
> I'm also open to suggestions for changing this, as long as we are consistent. cc @benolayinka

It looks good the way it is but is it documented anywhere?

I'll push a 'copy me' draft template to the docs with a skeleton of a docs page, and examples of these things. I can also add it to CONTRIBUTING.md if it isn't already somewhere else

Reply:
Documentation guidelines are here: https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/DEVELOPMENT.md